tests/package_tests/package_test.py

```diff
@@ -0,0 +1,220 @@
+#!/usr/bin/env python3
+
+# Copyright 2014-2021 Scalyr Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import pathlib
+import pathlib as pl
+import argparse
+import subprocess
+import json
+import time
+import os
+import tarfile
+import re
+
+
+def install_deb_package():
+    os.environ["LD_LIBRARY_PATH"] = f'/lib/x86_64-linux-gnu:{os.environ["LD_LIBRARY_PATH"]}'
+    subprocess.check_call(
+        ["dpkg", "-i", package_path],
+        env=os.environ,
+    )
+
+
+def install_rpm_package():
+    os.environ["LD_LIBRARY_PATH"] = "/libx64"
+    subprocess.check_call(
+        ["rpm", "-i", package_path],
+        env=os.environ,
+    )
+
+
+def install_tarball():
+    tar = tarfile.open(package_path)
+    tar.extractall(pathlib.Path("~").expanduser())
+    tar.close()
+
+def install_msi_package():
+    subprocess.check_call(
+        f"msiexec.exe /I {package_path} /quiet", shell=True
+    )
+
+
+
+
+def install_package(package_type: str):
+    if package_type == "deb":
+        install_deb_package()
+    elif package_type == "rpm":
+        install_rpm_package()
+    elif package_type == "tar":
+        install_tarball()
+    elif package_type == "msi":
+        install_msi_package()
+
+
+def determine_version():
+    if package_type in ["deb", "rpm"]:
```

Comment:
Hm, how do we determine version in this case?

One option would also be to consistently utilize version string from setup.py (which we can read from ``scalyr_agent/__init__.py`` ``__version__`` variable).